hack/test-system.sh

```diff
@@ -37,8 +37,12 @@ EOF
     export CONTAINERS_STORAGE_CONF=/etc/containers/storage.conf
 fi
 
-# Build skopeo, install into /usr/bin
-make PREFIX=/usr install
+# Print what binary is being used for tests
+if [[ -v SKOPEO_BINARY ]]; then
```

Comment:
(Absolutely non-blocking: Probably `-n "$…"`, because it is used in `${SKOPEO_BINARY:-…}` and that triggers on empty values as well. This should never make a difference in practice.)